pkg/util/util.go

```diff
@@ -715,6 +715,10 @@ func GetCertificateFromPEMBytes(bytes []byte) (*x509.Certificate, error) {
 	return cert, nil
 }
 
+func ReadFile(file string) ([]byte, error) {
+	return ioutil.ReadFile(file)
```

Comment:
This function has been deprecated since 1.16 and it is recommended to use `os.ReadFile`.

Reply:
right, package `ioutil` simply use package `os`,but there are too many `ioutil` in `fabric-operator` which needs to be updated.
I suggest we leave `iotuil` for now since `functionalities` doesn't change.